fe/fe-core/src/main/java/org/apache/doris/external/elasticsearch/EsNodeInfo.java

```diff
@@ -70,6 +70,7 @@ public EsNodeInfo(String id, Map<String, Object> map, boolean httpSslEnabled) {
         if (httpMap != null) {
             String address = (String) httpMap.get("publish_address");
             if (address != null) {
+                address = address.substring(address.lastIndexOf('/') + 1);
```

Comment:
add comments for this line, give an simple conf-example

Reply:
Provide UT-test for this case?

Reply:
added comments and unit tests for my fix. Please review. Thanks.